### DISCUSSION TITLE
[CQLd4] Sequential cql queries by the same thread

### DISCUSSION THREAD

**Discussion by chrispappo** [nosqlbench/nosqlbench#1333] [Q&A] [answered]
Hi,

With the CQLd4 driver I'm trying to find a way to execute multiple queries in the same cycle. 

Ops:
  insert: 
   prepared: 
     -write A 
    - write B

I want each thread to execute A and B sequentially  BUT with the same binding values. Any idea ?

**Comment by jshook** [comment, marked as answer]
@chrispappo , I apologize for not seeing this earlier.
To execute operations sequentially all you have to do is organize them that way in your workload template. If you want them to have the same binding values, then you can slow down the apparent rate of cycling by putting a `Div(5L)` where the number is the number of operations which should share the same bindings. This just means that the rest of each binding function will see the same input cycle, effectively, grouped by the number you provide.
For example, if you have 3 operations, and 7 binding functions among them, simply put `Div(3L)` in front of each of your 7 binding functions, and then your binding values will be the same for 3 cycles at a time.
